lib/hz2600/CallflowBuilder/Node/Voicemail.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace CallflowBuilder\Node;
+
+class Voicemail extends AbstractNode
+{
+    public function __construct($id) {
```

Comment:
`$id` should be optional; you can create a generic "check voicemail" callflow action where the caller must enter their box ID and pin.